.github/workflows/ci-build.yml

```diff
@@ -0,0 +1,46 @@
+# NOTE: This name appears in GitHub's Checks API and in workflow's status badge.
+name: ci-build
+
+# Trigger the workflow when:
+on:
+  # A push occurs to one of the matched branches.
+  push:
+    branches: [stable, master]
+  # Or when a pull request event occurs for a pull request against one of the
+  # matched branches.
+  pull_request:
+    branches: [stable, master]
+
+# disable secrets.GITHUB_TOKEN permissions
+permissions: {}
+
+jobs:
+  yarn_cache:
```

Comment:
Do we need this `yarn_cache` job here or just in the `ci-test` workflow?

Reply:
We don't